drivers/storage/portworx/status.go

```diff
@@ -79,15 +73,15 @@ func (p *portworx) UpdateStorageClusterStatus(
 	cluster.Status.ClusterName = pxCluster.Cluster.Name
 	cluster.Status.ClusterUID = pxCluster.Cluster.Id
 
-	return p.updateStorageNodes(clientConn, cluster)
+	return p.updateStorageNodes(p.sdkConn, cluster)
```

Comment:
nit: Might as well not pass `p.sdkConn` as we have access to `p.sdkConn` inside updateStorageNodes now.